engine/src/vs_globals.h

```diff
@@ -72,41 +98,41 @@ extern Mission    *mission;
 template < class MyType >
 class LeakVector
 {
-    std::vector< MyType > *active_missions;
+    std::vector< MyType > *internal_vector_p_;
```

Comment:
for clarity, this should be declared private (default for class) or protected.

Reply:
Done.